### ISSUE TITLE
Model initialization problem

### ISSUE THREAD

**Issue by yqw0710** [YacineGACI/AttenD#1]
Hello, I have read your paper and code. I have a question here. I used BertModel to initialize the model and extract Attention to calculate the loss to fine-tune the model when I tried it myself, but the score is different each time when evaluating the debiased model with crows-pair. I found that when using the debiased model to perform MLM tasks, each time prediction score is different. I saw that your code was also initialized with AutoModel, and I encountered the same problem after running your code. I want to ask you whether you have encountered this problem in the experiment and whether I have neglected some problems in the experiment. Thank you very much.

**Comment by YacineGACI** [comment]
Hello. I did not encounter this problem before, at least not with BERT model (I believe it was ELECTRA that gave non-deterministic results). 

Have you tried to finetune the debiased text encoder on a MLM objective? Since you are using Crows-Pairs experiment, which is based on language modeling likelihoods of masked tokens, you have to finetune on MLM before running the test. The same applies to StereoSet as well.

**Comment by yqw0710** [comment]
Thank you very much for your reply. I have not finetune debiased model on a masked language modeling (mlm) objective. Thanks!
